src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +62,25 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// content generated by function
+export function DynamicRowGeneration = (rowElementMaxCount, labelArray) => {
+  // todo: QUESTION - should I use generateLabelSpanContents before or after chunking?
+  // todo: QUESTION - should I apply use this function for the Applicant Information Section as well?
+  // todo: what happens when we have an odd number of elements in an array with an even maxRowElementsValue? do we need to generate a blank span?
+  //generateLabelSpanContents('', 'row-six-blank', '', false) //blank span to keep row elements in line with those above
+
+  // use the chunk method to organize them in arrays of two
+  const chunkedArr = chunk(rowElementMaxCount)(labelArray);
+
+  // use a map function to generate a new array that wraps each chunk in the row style
+  // template that you can then plug into the component's return statement
+  const output = chunkedArr.map(chunk => {
+    return div({className: 'information-row', style: styles.applicantInfoRow}, chunk);
+  };
```

Comment:
You need to close the parenthesis from the map function (so it needs to look like `});`